src/old/lib/utilities/API/types.ts

```diff
@@ -316,3 +316,15 @@ export type GetChangeLogType = ChangeLogType & {
   title?: string;
   totalPages: number;
 };
+
+export type CreateUpdateAuthorRequestType = {
```

Comment:
not a big deal, but I'd have 2 different types(like separate names) right from the start. 
Why it might be helpful - well, not here specifically, but in general case - 

1. when it's a type of argument we have better visibility what's passed and for what
2. again, not for this case, but it's quite often to have some fields can be set only once, per creation; so having CreateRequestType and UpdateRequestType right from the start means less headache in this scenario
3. while they are equal, sure, no need to completely duplicate whole the type definition; just using type aliases would be enough by now